tests/ClientFunctionalTest.php

```diff
@@ -174,8 +174,10 @@ public function testBulkIndex(): void
         $this->assertEquals('AnonCoin', $index->getDocument(1)->get('name'));
         $this->assertEquals('iXcoin', $index->getDocument(2)->get('name'));
 
-        $ixCoin->setIndex(null);  // Make sure the index gets set properly if missing
-        $index->deleteDocuments([$anonCoin, $ixCoin]);
+        $index->deleteDocuments([
+            new Document('1', ['name' => 'AnonCoin']),
```

Comment:
Why do you need to recreate the doc here for this change? I also wander why we do it onl line 170/171?

Reply:
I've updated the test to reuse the documents and added a couple of tests to check that the index is set if missing (to remove the `$ixCoin->setIndex(null);` part)